src/app/Library/CrudPanel/CrudPanel.php

```diff
@@ -362,20 +370,26 @@ private function getRelationModelInstances($model, $relationString)
         $relationArray = explode('.', $relationString);
```

Comment:
@pxpm I'm sure you're right, but why was this change needed? 😄

Reply:
Explained in previous comment. About the change in other function